### PR TITLE
Done for you gatsby demo re write #113

### PR DESCRIPTION
**Pull request by olavea**
## 2. check language:

2.1. intricacies of yet another -> fast moving [b15b2e1](https://github.com/raae/queen.raae.codes/pull/114/commits/b15b2e1b6e715e5c7683b60208a4290503648d5a)

### CODE REVIEW

File: content/landing/done-for-you-demo/_how.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ## Here's How It Works
 
-You let us know what comes up again and again in support requests and point us towards relevant documentation.
+You let us know what comes up in support requests and point us towards relevant documentation.
 
 We propose a feature set for the demo showcasing solutions to the most costly support request.
 
-You approve the feature set and decide on the demo framework (i.e., Gatsby, Next, or plain React).
+You approve the feature set and decide to get this Gatsby demo started.
```
Comment (raae) [inline content/landing/done-for-you-demo/_how.md:7]: Maybe:

You approve the feature set and **we** the Gatsby demo started.

File: content/landing/done-for-you-demo/_how.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ The base price is USD 4.500 paid in full before we start coding.
 #### Optional Extras
 
 - Styling with either MUI or Tailwind&nbsp;CSS&nbsp;+&nbsp;USD&nbsp;1.500
-- Additional framework (i.e. Gatsby, Next, or plain&nbsp;React)&nbsp;+&nbsp;USD&nbsp;1.500&nbsp;each
+- Additional framework (i.e. Next, or plain&nbsp;React)&nbsp;+&nbsp;USD&nbsp;1.500&nbsp;each
```
Comment (raae) [inline content/landing/done-for-you-demo/_how.md:21]: I like that you kept these in here!

**Review by raae — APPROVED** [review]

File: content/landing/done-for-you-demo/_how.md (hunk 1 above)
Comment (olavea) [reply content/landing/done-for-you-demo/_how.md]: You approve the feature set and we get the Gatsby demo started.

File: content/landing/done-for-you-demo/_how.md (hunk 2 above)
Comment (olavea) [reply content/landing/done-for-you-demo/_how.md:21]: 🏴‍☠️ 😺 👍

**Pull request merged**